observability/distr_tracing/distr-tracing-tempo-installing.adoc

```diff
@@ -86,7 +91,7 @@ include::modules/distr-tracing-tempo-install-gateway-write-permissions.adoc[leve
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../../observability/otel/otel-installing.adoc#install-otel[Creating the required RBAC resources automatically]
+* xref:../../observability/otel/otel-installing.adoc#install-otel[Creating the required RBAC resources automatically]
```

Comment:
This one links to "Installing"

Reply:
Same situation as in the previous thread.